lightkurve/seismology.py

```diff
@@ -0,0 +1,258 @@
+"""Defines the asteroseismology module"""
+from __future__ import division, print_function
+
+import copy
+import os
+import logging
+import warnings
+
+import numpy as np
+from matplotlib import pyplot as plt
+import astropy
+from astropy import units as u
+from astropy import constants as const
+
+from uncertainties import ufloat
+from uncertainties import umath
+
+log = logging.getLogger(__name__)
+
+__all__ = ['estimate_radius','estimate_mass','estimate_logg']
+
+"""Global parameters for the sun"""
+NUMAX_SOL = ufloat(3090, 30) # microhertz | Huber et al. 2011
+DNU_SOL = ufloat(135.1, 0.1) # microhertz | Huber et al. 2011
+TEFF_SOL = ufloat(5772., 0.8) # Kelvin    | Prša et al. 2016
```

Comment:
Oops, unfortunately the unicode `š` character is making the Lightkurve tests fail on Python 2.  Better replace it with an `s`.  Sorry Andrej!!
```suggestion
TEFF_SOL = ufloat(5772., 0.8) # Kelvin    | Prsa et al. 2016
```